### ISSUE TITLE
AA portable weapon suggestion

### ISSUE THREAD

**Issue by duckinrat** [UncreatedStaff/UncreatedWarfare#135]
Id like to see something like a FIM-92mStinger or Igla in the game, cuz if youve just placed an fob and some enemy guys took a jet you cant do a thing against them. Thanx!

**Comment by Samanthuhh** [comment]
NGL i agree, a javelin or portable aa of some sort, it would be vv helpful and would add strategy to armor players

**Comment by HamzaXSniper** [comment]
Why need a javelin when you already have anti tank (rpg and maaws) adding a guided one would just make this not fun for armored players. portable anti air would be also too op and you would be shot at from anywhere rather than fobs which are abit less you know **portable** .

**Comment by UsernameIsUnavailible** [comment]
For S3 AA & Flare changes are planned.

**Comment by HamzaXSniper** [comment]
Even if flares and aa will be changed a jet can be shot down by a person with this "Anti air kit" that it just flew pass and just thought it was an infantryman, It would simply not have enough time to react and would be hit.  Sure an attack heli might be a little bit more protected as the gunner can just shoot them like they would normally would but that leaves the multi-role jet exposed to this kind of stuff. If the "aa and flare changes" include a minimum lock on range for these portable anti airs it wouldnt be such a bad idea to add it. Am still not changing my mind about the javelin tho and i think alot can agree javelins require no skill.

**Comment by ToastyTimeZ** [comment]
Portable AA will never be added unless there is a limit made for building AA like only 1-2 AA allowed entirely for the teams, meaning you get 2 fobs that can have 1 AA and that's all

**Comment by UsernameIsUnavailible** [comment]
Mobile AA is planned, Kits with AA guns is not discussed yet/Not planned due to it being pretty hard to balance (e.g custom kits with AA, 3-4 AA kits would mean jets/helis stand NO chance at all).

**Comment by ToastyTimeZ** [comment]
Exactly, plus with that stuff, it needs to be made where AA doesn't just ignore flares that are right next to the aircraft just because they weren't deployed at that moment but before

**Comment by UsernameIsUnavailible** [comment]
Again, Flare changes are planned..

**Comment by ToastyTimeZ** [comment]
Ye ik

**Comment by benvalkin** [comment]
This would be nice to do at some point
